src/llmcompressor/modifiers/transform/quip/base.py

```diff
@@ -123,6 +130,46 @@ def on_finalize(self, state: State, **kwargs) -> bool:
 
         return True
 
+    def _untie_if_target_shared_embedding(self, model: torch.nn.Module):
```

Comment:
nit -- If this is the same as `QuantizationMixin._untie_if_target_shared_embedding`, might be worth moving to a shared util.